index.html

```diff
@@ -11,34 +11,217 @@
                 <p class="para">
                    I can help you build a product , feature or website Look through some of my work 
                    and experience! If you like what you see and have a project you need coded, 
                    don’t hestiate to contact me.
                </p>
-               <div>
                 <ul class="anchor">
                    <li><a href="#"><img class="social" src="./img/social icons/Icon -GitHub.png" alt="-GitHub-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Linkedin.png" alt="Linkedin-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Angelist.png" alt="Angelist-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Twitter.png" alt="Twitter-icon"></a></li>
                    <li><a href="#"><img class="social" src="./img/social icons/Icon - Medium.png" alt="Medium-icon"></a></li>
                 </ul>   
-               </div> 
             </div>
         </section>
-
         
-      
 
+        <div class="main-container">
+            <div class="tittle">
+                <h2 class="tittle-header">My Recent Works</h2>
+                <img class="vector" src="./img/work/Vector.svg" alt="Vector">
+            </div>
+            <div class="work-cards">
+                <div class="content-container">   
+                    <div class="card1-container">
+                        <img class="works-img" src="./img/work/Img Placeholder.svg" alt="image">
+                        <div class="card1-content">
+                            <h3 class="works-header">Multi-Post Stories</h3>
+                            <p class="works-txt">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                    
+                            <ul class="tech">
+                                    <li><a href="#" class="css">css</a></li>
+                                    <li><a href="#" class="html">html</a></li>
+                                    <li><a href="#" class="boot">bootstrap</a></li>
+                                    <li><a href="#" class="ruby">Ruby</a></li>
+                            </ul>
+                            <a href="#"><button class="pro1-text hover-effect">See Project</button></a>
+                        </div>                    
+                    </div>
+
+                    <div class="cards-frame">
+                        <div class="all-cards">            
+                                <div class="no-btn">
+                                    <h1>Profesional Art Printing Data</h1>
```

Comment:
Convention and HTML best practices state that only one `h1 tag` should be used on a webpage. You have already used the `h1 tag` in the Headline section. Kindly change this tag to another `hx tag` between `h2-h6.` 💻

## Remember:-

You did the same mistake in all the cards onwards so consider reviewing and correcting them also.